kimchi/src/oracles.rs

```diff
@@ -47,6 +47,7 @@ pub mod caml {
 
     use super::*;
 
+    #[derive(ocaml::IntoValue, ocaml::FromValue, ocaml_gen::Struct)]
```

Comment:
This may need to be feature flagged, but I'll assume that you `cargo build`-ed locally to confirm.

Reply:
we're inside `#[cfg(feature = "ocaml_types")]pub mod caml { ... }`